Reference/Config/umbracoSettings/index.md

```diff
@@ -11,20 +11,20 @@ Below you can see the settings that affect the Umbraco backoffice (v7.8+)
 
 ### Tours
 
-The section is used for controlling the backoffice tours functionality
+The section is used for controlling the backoffice tours functionality.
 
 	<backOffice>
         <tours enable="true"></tours>
     </backOffice>
 
-There is only one supported attribute on the tours element:
+There is only one supported attribute on the tours element :
```

Comment:
Should there be a space here?

generally in the docs we prefer the colon next to the word and without a space...

Reply:
Yes, got this.
Fixed that.